webpack-module.config.js

```diff
@@ -1,5 +1,7 @@
 // module config for webpack
-module.exports = (nodeEnv) => ({
+module.exports = (nodeEnv) => nodeEnv === 'development' ?
+{} :
+({
```

Comment:
this makes it so that if we're running in development, just pass in an empty object. if we're in production, actually do the transforms